tests/conftest.py

```diff
@@ -386,8 +385,21 @@ async def postgresql_client(postgres_db, database_name_internal):
     await client.close()
 
 
+@pytest.fixture(scope="function")
+async def postgresql_pool(postgres_db, database_name_internal):
+    pool = await asyncpg.create_pool(
+        host=postgres_db.host,
+        port=postgres_db.port,
+        user=postgres_db.user,
+        password=postgres_db.password,
+        database=database_name_internal,
+    )
+    yield pool
+    await pool.close()
+
+
 @pytest.fixture
-def sqlalchemy_url_parameters(postgres_db, database_name_internal: str) -> dict[str, str]:
+def sqlalchemy_url_parameters(postgres_db, database_name_internal: str, postgresql_pool) -> dict[str, str]:
```

Comment:
Can't we now remove this one and the `sql_alchemy_engine` fixture. Are they still used?